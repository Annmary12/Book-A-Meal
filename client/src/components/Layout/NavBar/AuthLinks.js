import React, { Fragment } from 'react';
import CustomerLinks from './CustomerLinks.js';
import { Link } from 'react-router-dom';

const AuthLinks = ({ role, pathname }) => {
  //if (role === 'customer') {
    return (
      <CustomerLinks
        pathname={pathname}
      />
    );
  //}
};

export default AuthLinks;
