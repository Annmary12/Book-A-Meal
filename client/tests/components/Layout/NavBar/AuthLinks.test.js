import React from 'react';
import AuthLinks from '../../../../src/components/Layout/NavBar/AuthLinks';

describe('AuthLinks component', () => {
  it('should match snapshot if role is customer', () => {
    const wrapper = shallow(
      <AuthLinks
        role="customer"
        pathname="test"
      />
    );

    expect(wrapper).toMatchSnapshot();
  })

  it('should match snapshot if role is not customer', () => {
    const wrapper = shallow(
      <AuthLinks
        role="caterer"
        pathname="test"
      />
    );

    expect(wrapper).toMatchSnapshot();
  })
});