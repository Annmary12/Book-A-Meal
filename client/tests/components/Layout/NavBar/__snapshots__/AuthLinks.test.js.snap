// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthLinks component should match snapshot if role is customer 1`] = `
<CustomerLinks
  pathname="test"
/>
`;

exports[`AuthLinks component should match snapshot if role is not customer 1`] = `
<CatererLinks
  pathname="test"
/>
`;
